stackit/internal/services/loadbalancer/loadbalancer/resource.go

```diff
@@ -446,6 +457,18 @@ The example below uses OpenStack to create the network, router, a public IP addr
 							Description: descriptions["target_port"],
 							Required:    true,
 						},
+						"session_persistence": schema.SingleNestedAttribute{
+							Description: descriptions["session_persistence"],
```

Comment:
this description is missing from the `descriptions` map

Reply:
done